storage-provider/server/src/config.rs

```diff
@@ -42,6 +42,13 @@ fn default_node_address() -> Url {
     Url::parse(DEFAULT_NODE_ADDRESS).expect("DEFAULT_NODE_ADDRESS must be a valid Url")
 }
 
+fn default_retrieval_address() -> Multiaddr {
+    "/ip4/127.0.0.1/tcp/8002"
+        .to_string()
+        .parse()
+        .expect("multiaddres is correct")
```

Comment:
```suggestion
    "/ip4/127.0.0.1/tcp/8002"
        .parse()
        .expect("multiaddres is correct")
```
[No need to convert to string first.](https://docs.rs/libp2p/latest/libp2p/struct.Multiaddr.html#impl-TryFrom%3C%26str%3E-for-Multiaddr)